src/Search_Replace_Command.php

```diff
@@ -595,15 +597,22 @@ static function ( $key ) {
 						$update_where[ $k ] = $v;
 					}
 
-					$updates[] = [ $table, array( $col => $value ), $update_where ];
+					$wpdb->update( $table, [ $col => $value ], $update_where );
 				}
 			}
 
-			$offset += $limit;
-		}
-
-		foreach ( $updates as $update ) {
-			$wpdb->update( ...$update );
+			// Because we are ordering by primary keys from least to greatest,
```

Comment:
Love it, 2 wins. Runs faster than limit/offset and solves the memory problems. Thanks for working on it. 🙇🏻